src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -0,0 +1,208 @@
+      const ::Strahlkorper<Frame>& strahlkorper) noexcept {
+    return strahlkorper.ylm_spherepack().spec_to_phys(
+        strahlkorper.coefficients());
+  }
+  using argument_tags = typelist<Strahlkorper<Frame>>;
+};
+
+/// \f$(x,y,z)\f$ of each point on the surface.
+template <typename Frame>
+struct CartesianCoords : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "CartesianCoords";
+  static typename types<Frame>::Vector function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::OneForm& r_hat) noexcept;
+  using argument_tags =
+      typelist<Strahlkorper<Frame>, Radius<Frame>, Rhat<Frame>>;
+};
+
+/// dx_radius(i) is \f$\partial r/\partial x^i\f$.
+/// Here \f$r=r(\theta,\phi)\f$ is the surface, which is considered a
+/// function of Cartesian coordinates \f$r=r(\theta(x,y,z),\phi(x,y,z))\f$
+/// for this operation.
+template <typename Frame>
+struct DxRadius : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "DxRadius";
+  static typename types<Frame>::OneForm function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::InvJacobian& inv_jac) noexcept;
+  using argument_tags =
+      typelist<Strahlkorper<Frame>, Radius<Frame>, InvJacobian<Frame>>;
+};
+
+/// d2x_radius(i,j) is \f$\partial^2 r/\partial x^i\partial x^j\f$.
+/// Here \f$r=r(\theta,\phi)\f$ is the surface, which is considered a
+/// function of Cartesian coordinates \f$r=r(\theta(x,y,z),\phi(x,y,z))\f$
+/// for this operation.
+template <typename Frame>
+struct D2xRadius : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "D2xRadius";
+  static typename types<Frame>::SecondDeriv function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::InvJacobian& inv_jac,
+      const typename types<Frame>::InvHessian& inv_hess) noexcept;
+  using argument_tags = typelist<Strahlkorper<Frame>, Radius<Frame>,
+                                 InvJacobian<Frame>, InvHessian<Frame>>;
+};
+
+/// \f$\nabla^2 radius\f$, the flat Laplacian of the surface.
+/// This is \f$\eta^{ij}\partial^2 r/\partial x^i\partial x^j\f$,
+/// where \f$r=r(\theta(x,y,z),\phi(x,y,z))\f$.
+template <typename Frame>
+struct NablaSquaredRadius : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "NablaSquaredRadius";
+  static DataVector function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::ThetaPhi& theta_phi) noexcept;
+  using argument_tags =
+      typelist<Strahlkorper<Frame>, Radius<Frame>, ThetaPhi<Frame>>;
+};
+
+/// Cartesian components of (unnormalized) one-form defining the surface.
+/// This is computed by \f$x_i/r-\partial r/\partial x^i\f$,
+/// where \f$x_i/r\f$ is `r_hat` and
+/// \f$\partial r/\partial x^i\f$ is `dx_radius`.
+/// See Eq. (8) of arXiv:gr-qc/9606010
```

Comment:
It would be super convenient to actually have a hyper link to the arxiv: `[arXiv:gr-qc/eee](the_link_to_the_arxiv)`